app/views/roles/_roles_table.html.erb

```diff
@@ -1,23 +1,47 @@
-<div class="table-responsive">
-  <table class="table table-hover table-striped">
-    <thead>
-      <tr>
-        <th>nombre</th>
-        <th>Descripcion</th>
-        <th>Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% roles.each do |role| %>
-        <tr>
-          <td><%= role.name %></td>
-          <td><%= role.description %></td>
-          <td>
-            <%= link_to '<span class="glyphicon glyphicon-pencil"></span>'.html_safe, edit_role_path(role), class: "btn btn-default btn-sm", data: { "toggle" => "tooltip" }, title: "Edit" %>
-        		<%= link_to 'Eliminar', role_path(role), method: :delete %>
-          </td>
-        </tr>
-      <% end %>
-    </tbody>
-  </table>
+<div class="content">
+  <div class="material-datatables">
+    <div class="dataTables_wrapper container-fluid dt-bootstrap4">
+      <div class ="row">
```

Comment:
The structure of the html should be 
<div class="content">
   <div class="material-datatables">
     <table id="roles-table" class="....">

Also, you should use a '-' instead of an underscore in the names of the ids and classes in the HTML.